docs/MAINTAINERS.txt

```diff
@@ -38,3 +38,4 @@ Maintainers:
   Joshua Lock
     Email: jlock@vmware.com
     GitHub username: @joshuagl
+    PGP fingerprint: 08F3 409F CF71 D87E 30FB D3C2 1671 F65C B748 32A4
```

Comment:
Do you have a Keybase profile we can use to verify?

Reply:
I've just signed up, verified in a couple of places and updated this patch to add the username.